packages/aws-appsync-subscription-link/src/realtime-subscription-handshake-link.ts

```diff
@@ -607,7 +608,7 @@ export class AppSyncRealTimeSubscriptionHandshakeLink extends ApolloLink {
             } = data;
             if (type === MESSAGE_TYPES.GQL_CONNECTION_ACK) {
               ackOk = true;
-              this.keepAliveTimeout = connectionTimeoutMs;
+            this.keepAliveTimeout = this.keepAliveTimeout ?? connectionTimeoutMs;
```

Comment:
Different than
```
this.keepAliveTimeout ??= connectionTimeoutMs;
```
?

Reply:
Same thing, but only supported in typescript `>=4.0`, we are currently on `~3.8`